parseAndPopulate/resolve_expiration.py

```diff
@@ -55,10 +51,8 @@ def __init__(self):
         super().__init__(help, None, None if __name__ == '__main__' else [])
 
 
-def main(scriptConf=None):
+def main(script_conf: t.Optional[BaseScriptConfig] = None):
```

Comment:
If we never initialize a `ScriptConfig` object the `--help` argument will be ignored.

Reply:
thanks, fixed